src/esockd_udp.erl

```diff
@@ -29,13 +29,11 @@
 -export([ get_options/1
         , get_acceptors/1
         , get_max_connections/1
-        , get_max_conn_rate/3
```

Comment:
What is the reason we need to remove the `get_max_conn_rate/3` and `set_max_conn_rate/4` functions?